### ISSUE TITLE
Event where all the subCats have been spliced

### ISSUE THREAD

**Issue by Deohgu** [Deohgu/random-wikipedia-of#15]
ON HOLD. Depends on Issue #14

Currently to add more variety of content, each time the user requests another category of the original fetch, the code picks a random sub category, fetches its articles & subCats, and adds it to the available list of possible articles to get.
But when a new sub category is picked to to search the code splices that sub category in order to not accidently have it picked and duplicate the existing articles of that sub cat.

### Issue :
- When all the subCats have been spliced out the Web App is likely to throw an error as there is not new sub cats to pick from.
Testing is needed to affirm this bug.
Structure of fetching is likely to change as well.
